Makefile

```diff
@@ -332,7 +332,7 @@ push-image: build-image
 
 .PHONY: local
 ## Local: Run operator locally
-local: deploy-clean deploy-rbac deploy-crds
+local: deploy-crds deploy-clean deploy-rbac
```

Comment:
@Avni-Sharma Why did you change this order?

Reply:
@DhritiShikhar if you run `deploy-crds` target first and then `deploy-clean` it will delete the crds deployed by `deploy-crds` target. I think there isn't a problem with the original sequence. WDYT?

Reply:
There are errors generated because of this change

```
➜  service-binding-operator git:(consistent-crds) make local
cd ./deploy/crds/ && ln -srf ../../manifests-upstream/0.0.20/servicebindingrequests.apps.openshift.io.crd.yaml \
apps_v1alpha1_servicebindingrequest_crd.yaml
cd ./deploy/crds && ln -srf ../../examples/nodejs_postgresql/service-binding-request.yaml \
apps_v1alpha1_servicebindingrequest_cr.yaml
kubectl create -f deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml
Error from server (AlreadyExists): error when creating "deploy/crds/apps_v1alpha1_servicebindingrequest_crd.yaml": customresourcedefinitions.apiextensions.k8s.io "servicebindingrequests.apps.openshift.io" already exists
make: *** [Makefile:348: deploy-crds] Error 1
```

Reply:
@pratikjagrut Yes. I agree with you.

@Avni-Sharma please remove this change.